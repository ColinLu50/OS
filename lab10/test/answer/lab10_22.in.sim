4993
267
271
145
263
140
