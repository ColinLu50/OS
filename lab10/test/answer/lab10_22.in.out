FCFS
126 -> 100 -> 86 -> 52 -> 47 -> 133 -> 110 -> 29 -> 88 -> 85 -> 94 -> 23 -> 5 -> 127 -> 95 -> 59 -> 110 -> 30 -> 51 -> 140 -> 91 -> 4 -> 102 -> 136 -> 137 -> 87 -> 52 -> 145 -> 45 -> 124 -> 114 -> 123 -> 128 -> 121 -> 30 -> 45 -> 33 -> 24 -> 134 -> 120 -> 39 -> 32 -> 49 -> 37 -> 133 -> 4 -> 100 -> 66 -> 80 -> 97 -> 137 -> 107 -> 140 -> 39 -> 107 -> 9 -> 40 -> 7 -> 117 -> 141 -> 126 -> 99 -> 125 -> 76 -> 46 -> 105 -> 4 -> 19 -> 123 -> 27 -> 103 -> 26 -> 124 -> 25 -> 136 -> 135 -> 78 -> 128 -> 4 -> 92 -> 55 -> 136 -> 57 -> 114 -> 113 -> 57 -> 101 -> 95 -> 36 -> 117 -> 80 -> 60 -> 53 -> 10 -> 134 -> 50 -> 15
4993
SSTF
126 -> 126 -> 127 -> 128 -> 128 -> 125 -> 124 -> 124 -> 123 -> 123 -> 121 -> 120 -> 117 -> 117 -> 114 -> 114 -> 113 -> 110 -> 110 -> 107 -> 107 -> 105 -> 103 -> 102 -> 101 -> 100 -> 100 -> 99 -> 97 -> 95 -> 95 -> 94 -> 92 -> 91 -> 88 -> 87 -> 86 -> 85 -> 80 -> 80 -> 78 -> 76 -> 66 -> 60 -> 59 -> 57 -> 57 -> 55 -> 53 -> 52 -> 52 -> 51 -> 50 -> 49 -> 47 -> 46 -> 45 -> 45 -> 40 -> 39 -> 39 -> 37 -> 36 -> 33 -> 32 -> 30 -> 30 -> 29 -> 27 -> 26 -> 25 -> 24 -> 23 -> 19 -> 15 -> 10 -> 9 -> 7 -> 5 -> 4 -> 4 -> 4 -> 4 -> 133 -> 133 -> 134 -> 134 -> 135 -> 136 -> 136 -> 136 -> 137 -> 137 -> 140 -> 140 -> 141 -> 145
267
SCAN
126 -> 125 -> 124 -> 123 -> 121 -> 120 -> 117 -> 114 -> 113 -> 110 -> 107 -> 105 -> 103 -> 102 -> 101 -> 100 -> 99 -> 97 -> 95 -> 94 -> 92 -> 91 -> 88 -> 87 -> 86 -> 85 -> 80 -> 78 -> 76 -> 66 -> 60 -> 59 -> 57 -> 55 -> 53 -> 52 -> 51 -> 50 -> 49 -> 47 -> 46 -> 45 -> 40 -> 39 -> 37 -> 36 -> 33 -> 32 -> 30 -> 29 -> 27 -> 26 -> 25 -> 24 -> 23 -> 19 -> 15 -> 10 -> 9 -> 7 -> 5 -> 4 -> 0 -> 127 -> 128 -> 133 -> 134 -> 135 -> 136 -> 137 -> 140 -> 141 -> 145
271
C-SCAN
126 -> 125 -> 124 -> 123 -> 121 -> 120 -> 117 -> 114 -> 113 -> 110 -> 107 -> 105 -> 103 -> 102 -> 101 -> 100 -> 99 -> 97 -> 95 -> 94 -> 92 -> 91 -> 88 -> 87 -> 86 -> 85 -> 80 -> 78 -> 76 -> 66 -> 60 -> 59 -> 57 -> 55 -> 53 -> 52 -> 51 -> 50 -> 49 -> 47 -> 46 -> 45 -> 40 -> 39 -> 37 -> 36 -> 33 -> 32 -> 30 -> 29 -> 27 -> 26 -> 25 -> 24 -> 23 -> 19 -> 15 -> 10 -> 9 -> 7 -> 5 -> 4 -> 0 -> 146 -> 145 -> 141 -> 140 -> 137 -> 136 -> 135 -> 134 -> 133 -> 128 -> 127
145
LOOK
126 -> 125 -> 124 -> 123 -> 121 -> 120 -> 117 -> 114 -> 113 -> 110 -> 107 -> 105 -> 103 -> 102 -> 101 -> 100 -> 99 -> 97 -> 95 -> 94 -> 92 -> 91 -> 88 -> 87 -> 86 -> 85 -> 80 -> 78 -> 76 -> 66 -> 60 -> 59 -> 57 -> 55 -> 53 -> 52 -> 51 -> 50 -> 49 -> 47 -> 46 -> 45 -> 40 -> 39 -> 37 -> 36 -> 33 -> 32 -> 30 -> 29 -> 27 -> 26 -> 25 -> 24 -> 23 -> 19 -> 15 -> 10 -> 9 -> 7 -> 5 -> 4 -> 127 -> 128 -> 133 -> 134 -> 135 -> 136 -> 137 -> 140 -> 141 -> 145
263
C-LOOK
126 -> 125 -> 124 -> 123 -> 121 -> 120 -> 117 -> 114 -> 113 -> 110 -> 107 -> 105 -> 103 -> 102 -> 101 -> 100 -> 99 -> 97 -> 95 -> 94 -> 92 -> 91 -> 88 -> 87 -> 86 -> 85 -> 80 -> 78 -> 76 -> 66 -> 60 -> 59 -> 57 -> 55 -> 53 -> 52 -> 51 -> 50 -> 49 -> 47 -> 46 -> 45 -> 40 -> 39 -> 37 -> 36 -> 33 -> 32 -> 30 -> 29 -> 27 -> 26 -> 25 -> 24 -> 23 -> 19 -> 15 -> 10 -> 9 -> 7 -> 5 -> 4 -> 145 -> 141 -> 140 -> 137 -> 136 -> 135 -> 134 -> 133 -> 128 -> 127
140
